indico/modules/events/registration/util.py

```diff
@@ -995,3 +997,22 @@ def get_persons(registrations, include_accompanying_persons=False):
                             'registration': registration,
                             'is_accompanying': True} for person in registration.accompanying_persons)
     return persons
+
+
+def resize_uploaded_registration_picture(source, size):
+    """Resize uploaded registration picture to the given size."""
+    try:
+        picture = Image.open(source)
+        picture = ImageOps.exif_transpose(picture)
+        size_x, size_y = picture.size
+        if max(size_x, size_y) < size:
+            source.seek(0)
+            return source
+        ratio = float(size) / max(size_x, size_y)
```

Comment:
Is the explicit cast to float needed?
```suggestion
        ratio = size / max(size_x, size_y)
```

Reply:
this is removed